testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvMultinodeKafkaSaslPlaintext.java

```diff
@@ -40,38 +38,37 @@
     private final ResourceProvider configDir;
 
     @Inject
-    public EnvMultinodeKafkaSaslPlaintext(KafkaSaslPlaintext kafka, StandardMultinode standardMultinode, DockerFiles dockerFiles)
+    public EnvMultinodeKafkaSaslPlaintext(KafkaSaslPlaintext kafka, MultinodeProvider multinodeProvider, DockerFiles dockerFiles)
     {
-        super(standardMultinode, kafka);
+        super(multinodeProvider.singleWorker(), kafka);
         requireNonNull(dockerFiles, "dockerFiles is null");
         configDir = dockerFiles.getDockerFilesHostDirectory("conf/environment/multinode-kafka-sasl-plaintext/");
```

Comment:
It would be nice if we always or never used this - `configDir` is (sometimes) setup to reference the "root" of the particular Environment's configuration, so the `extendEnvironment` call can simply say `configDir.getPath(config.properties)` without having to specify the full path each time.